lib/puppet/indirector/face.rb

```diff
@@ -37,13 +37,15 @@ def self.terminus_classes(indirection)
     Puppet::Indirector::Terminus.terminus_classes(indirection.to_sym).collect { |t| t.to_s }.sort
   end
 
-  def call_indirection_method(method, key, options)
+  def call_indirection_method(method, key)
```

Comment:
I think it's probably safer to keep this method as-is and just pass through the `options` argument.